src/month.rs

```diff
@@ -42,29 +41,29 @@ use crate::OutOfRange;
 #[cfg_attr(all(feature = "arbitrary", feature = "std"), derive(arbitrary::Arbitrary))]
 pub enum Month {
     /// January
-    January = 0,
+    January = 1,
```

Comment:
Hmm, should we also make this explicitly `repr(u8)` then?

Reply:
As I understand it the only extra thing `repr(u8)` on an enum allows is unsafe pointer casting.

Reply:
Maybe, but to me it's also like a signal to the users about how this type works?

Reply:
With `repr(u8)` it no longer has niche optimization, but I suppose there is little use for that.

Reply:
Okay, I would actually say that's a good reason not to add it.